_posts/2020-06-20-Some-Examples-of-Querying.md

```diff
@@ -0,0 +1,304 @@
+---
+layout: post
+date: 2020-06-20 18:00:00
+title: Some Examples of Querying and More on the Cíannachta Glinne Geimen
+author: et
+---
+
+{% assign author = site.data.authors[page.author]%}
+{{author.name}}
+
+ORCID: <a href="https://orcid.org/{{ author.orcid }}" title="{{author.name}}">{{author.orcid}}</a>
+
+## Introduction
+In our [previous blogpost](https://cyocum.github.io/2020/06/06/Manuscript-Variants-and-the-Origins-of-C%C3%ADannachta-Glinne-Geimen.html), we looked at the variants between and within manuscript versions of the medieval Irish genealogies that we can encounter while working on IrishGen and how we go about encoding the information involved. This was in the context of the genealogies' account of the origins of the kingdom of the Cíannachta Glinne Geimen. In this post, we're going to continue following the history of this kingdom but, this time, consider some basic examples of how the IrishGen database can be queried.
+
+For introductory reading on the genealogies, we recommend Nollaig Ó Muraíle’s published lecture, _[The Irish Genealogies: Irish History’s Poor Relation?](https://www.ria.ie/its-occasional-lecture-vol-3-irish-genealogies-irish-historys-poor-relation)_, and Matthew Holmberg’s PhD thesis, ‘[Towards a Relative Chronology of the Milesian Genealogical Scheme](https://dash.harvard.edu/handle/1/37945002)’ (pp. 3–54). For an overview of IrishGen and our use of RDF, see the project [README](https://github.com/cyocum/irish-gen); for more in-depth discussion, see Christopher Yocum's previous blogposts on [linked data](https://cyocum.github.io/2020/06/17/IrishGen-RDF-Linked-Data-Semantic-Web.html) and [digital datasets](https://cyocum.github.io/2020/06/07/Human-Curation-and-Digital-Datasets.html). All our data is available in our [GitHub repository](https://github.com/cyocum/irish-gen).
```

Comment:
I am sorry to be a pest but if you could change `[linked data](https://cyocum.github.io/2020/06/17/IrishGen-RDF-Linked-Data-Semantic-Web.html)` to `[linked data]({% post_url 2020-06-17-IrishGen-RDF-Linked-Data-Semantic-Web %})` and `[digital datasets](https://cyocum.github.io/2020/06/07/Human-Curation-and-Digital-Datasets.html)` to `[digital datasets]({% post_url 2020-06-07-Human-Curation-and-Digital-Datasets %})`  This will ensure that if we ever move the blog, the URLs will still work correctly.

Reply:
No probs. Done.